### DISCUSSION TITLE
using student ID apply a permit before and delete the account, how to reapply????

### DISCUSSION THREAD

**Discussion by AaronZhai331** [community/community#64100] [GitHub Education]
### Hi everyone,

Hi! I used my student ID apply a permit before and deleted the account later. now I want to use my new email to apply, but it said I have used this student ID already. How could I apply again?

**Comment by christophercouch** [comment]
Hi @AaronZhai331,

If you've previously used your student ID for a GitHub Education benefit and later deleted the account, you'll need to contact GitHub Support directly. They can assist in verifying your details and resolving the issue with your student ID.

1. Go to GitHub's Contact Support page: https://support.github.com/contact
2. Choose "Account" as the subject.
3. Describe your issue, mentioning that you previously used your student ID for GitHub Education benefits but deleted the account and now wish to reapply.

Hope this helps!

**Comment by AliYa-chen** [comment]
Hi @AaronZhai331 
You go to GitHub's contact support page“ https://support.github.com/contact Choose 'education' as the theme
Describe your issue and mention that you previously received GitHub Education benefits using your student ID, but deleted the account and now wish to reapply.
Due to changes in the conditions for obtaining GitHub Education, it may be a bit troublesome. My work order for revalidating academia has been submitted for a week and I have not received a response.
